test/CDB_CartodbfyTableTest_expect

```diff
@@ -102,5 +102,12 @@ Table with non unique and null cartodb_id #148 cartodbfied fine
 3|
 4|2
 DROP TABLE
+CREATE TABLE
+
+DROP TABLE
+CREATE TABLE
+
+_cartodb_id0
+DROP TABLE
```

Comment:
I'd set the verbosity to NOTICE in the tests above to make sure the message "Could not initialize cartodb_id with existing values" is not raised during the tests.